src/dr_llm/llm/providers/google/reasoning.py

```
@@ -108,7 +109,7 @@ def _validate_google_reasoning_shape(
         )
         return
     raise ValueError(
-        f"google reasoning is not supported for provider='google' model={model!r}"
+        f"{ProviderName.GOOGLE} reasoning is not supported for provider='{ProviderName.GOOGLE}' model={model!r}"
     )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)